docs/using.rst

```diff
@@ -105,47 +113,57 @@ To use ADB with Host-Based tools
    3. `config.servicemanager.openshift_image_tag = "v1.1.1"` - Specifies the version of the image to be used.
 
-
 4. Start the ADB using ``vagrant up``. For details consult the `Installation documentation`_.
 
 .. _Installation documentation: https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst
 
-5. Configure the environment and download the required TLS certificates using the plugin.
-   The example below shows the command and the output for Linux and Mac OS X. On Microsoft Windows the output may vary depending on the execution environment.::
-
-    	$ vagrant service-manager env
-    	Configured services:
-    	docker - running
-        openshift - stopped
-    	kubernetes - stopped
-        docker env:
-    	# Set the following environment variables to enable access to the
-    	# docker daemon running inside of the vagrant virtual machine:
-    	export DOCKER_HOST=tcp://172.28.128.182:2376
-    	export DOCKER_CERT_PATH=/home/pchandra/test_adb/.vagrant/machines/default/libvirt/docker
-    	export DOCKER_TLS_VERIFY=1
-    	export DOCKER_API_VERSION=1.20
-    	# run following command to configure your shell:
-    	# eval "$(vagrant service-manager env docker)"
+
+5. Configure the environment and download the required TLS certificates using
+   the plugin. The example below shows the command and the output for Linux and Mac OS X.
+   On Microsoft Windows the output may vary depending on the execution environment::
+
+     $ vagrant service-manager env
+     # docker env:
+     # Set the following environment variables to enable access to the
+     # docker daemon running inside of the vagrant virtual machine:
+     export DOCKER_HOST=tcp://10.1.2.2:2376
+     export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+     export DOCKER_TLS_VERIFY=1
+     export DOCKER_API_VERSION=1.21
+
+     # run following command to configure your shell:
+     # eval "$(vagrant service-manager env)"
 
    Setting these environment variables allows programs, such as Eclipse and the
    docker CLI to access the docker daemon.
 
+   **Note:** When the OpenShift service is running in the VM, a docker registry
+   is also configured in the VM. This Docker registry can be consumed by external
+   tools such as Eclipse to push or pull images. The Docker registry url is exported
+   as a variable, and can be accessed as shown below::
+
+     $ vagrant service-manager env openshift
+     # openshift env:
+     # You can access the OpenShift console on: https://10.1.2.2:8443/console
+     # To use OpenShift CLI, run: oc login https://10.1.2.2:8443
+     export OPENSHIFT_URL=https://10.1.2.2:8443
+     export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
+     export DOCKER_REGISTRY=hub.openshift.centos7-adb.10.1.2.2.xip.io
+
 6. Begin developing.
 
    If you are using the docker CLI, you can just run it from the command line
```

Comment:
I just noticed this line is little inappropriate.
```
If you need to download a copy of the docker CLI, you can find it listed as a "client binary" download in the official Docker Repositories.
```
As, I couldn't find "client binary" phrase in the link mentioned.
Probably it could be changed as:
```
If you need to download a copy of the docker CLI, you can visit the [official Docker releases page](https://github.com/docker/docker/releases), go to the desired version(or click on `Tags` and choose version) and under "Downloads" section, download the desired OS based CLI.
```
Please rephrase above lines if you would like to :smile: